### PR TITLE
Template service - template dropdown

### PR DESCRIPTION
**Pull request by kanika1894**

### CODE REVIEW

File: src/configure/model/models.ts [hunk 1]
```diff
@@ -23,6 +24,18 @@ class ExtensionVariables implements UIExtensionVariables {
 let extensionVariables = new ExtensionVariables();
 export { extensionVariables };
 
+export class ApplicationSettings {
+    language: string;
+    buildTargetName: string;
+    deployTargetName: string;
+    workingDirectory: string;
+    settings?: Dictionary<string>;
```
[the remark this answers is not in the document]
Comment (kanika1894) [reply src/configure/model/models.ts]: done

File: src/configure/configure.ts [hunk 2]
```diff
@@ -412,29 +413,69 @@ class Orchestrator {
         //var repoAnalysisHelper = new RepoAnalysisHelper(this.inputs.azureSession);
         var repoAnalysisResult = null;
         //await repoAnalysisHelper.getRepositoryAnalysis(this.inputs.sourceRepository);
+        var templateServiceEnabled = false;
```
[the remark this answers is not in the document]
Comment (kanika1894) [reply src/configure/configure.ts]: done

File: src/configure/clients/TemplateServiceClient.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { RepoInfo } from "../model/models";
```
[the remark this answers is not in the document]
Comment (kanika1894) [reply src/configure/clients/TemplateServiceClient.ts]: done

File: src/configure/configure.ts [hunk 4]
```diff
@@ -412,29 +413,69 @@ class Orchestrator {
...
+                {
+                    "language": "Docker",
+                    "buildTargetName": "Dockerfile",
+                    "deployTargetName": "Azure:AKS",
+                    "workingDirectory": "wddddddddddddd"
+                }
+            ]
+        };
+
+        let serviceClient = new TemplateServiceClient();
```
[the remark this answers is not in the document]
Comment (kanika1894) [reply src/configure/configure.ts]: updated

**Comment by bishal-pdMSFT** [comment]
@kanika1894 anything blocking the merge here? Is it the scan failures?

**Pull request merged**